kratos/input_output/vtu_output.cpp

```diff
@@ -0,0 +1,259 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher (based on vtk_output.h)
```

Comment:
It is quite similar, could be done with a template?

Reply:
or may be a base class .. something like ParaviewOutputBaseClass

Reply:
I will probably refactor common functionalities into free functions. Even now there are many fct that don't require any class members